routes/api.php

```diff
@@ -48,7 +48,7 @@
 });
 
 // v3 routes
-$router->group(['prefix' => 'api/v3', 'middleware' => ['guard:api']], function () {
+$router->group(['prefix' => 'api/v3', 'middleware' => ['guard:api', 'scopes:activity']], function () {
```

Comment:
I think I slightly prefer attaching this `scopes:activity` in each of the controllers, so it's more easily discoverable (alongside where we require other scopes, authorization, or roles).

Reply:
makes sense! Updated :)